release.md

```diff
@@ -5,7 +5,7 @@
 To make a new release you can make use of the following `make` task.
 
 ```bash
-make gh-release NEW_VERSION=v0.6.0 OLD_VERSION=v0.5.0 DESCRIPTION="A test release to see how it works"
+make gh-release NEW_VERSION=v0.6.0 OLD_VERSION=v0.5.1-draft DESCRIPTION="A test release to see how it works"
```

Comment:
Add additional documentation here that explains what -draft tag does (that it triggers different goreleaser config)

Reply:
this is the `sed` replacement that also took this one. Will remove that commit before merge as there is no need to keep this release tryout in history.

Will make new PR with docs update for draft releases.